content/applications/finance/documents.rst

```diff
@@ -9,47 +9,119 @@ various workspaces.
 
 .. seealso::
    - `Odoo Documents: product page <https://www.odoo.com/app/documents>`_
+   - `Odoo Tutorials: Documents basics <https://www.odoo.com/slides/slide/documents-basics-674>`_
+   - `Odoo Tutorials: Using Documents with your Accounting App <https://www.odoo.com/slides/slide/using-documents-with-your-accounting-app-675?fullscreen=1#>`_
```

Comment:
Lines over 100 characters are hard to maintain (read, modify, review, ...).
```suggestion
   - `Odoo Tutorials: Using Documents with your Accounting App
     <https://www.odoo.com/slides/slide/using-documents-with-your-accounting-app-675?fullscreen=1#>`_
```

Reply:
when I split in two lines, I have an error: WARNING: Inline interpreted text or phrase reference start-string without end-string. 
![image](https://user-images.githubusercontent.com/85342080/201926661-4aa85488-3672-4af0-99bd-41031e68e3e5.png)

Reply:
Try removing the white space before `<https...`. The continuation line should be aligned with the **first character** of the line above, not with the first *alphabetical* character.

Reply:
solved!